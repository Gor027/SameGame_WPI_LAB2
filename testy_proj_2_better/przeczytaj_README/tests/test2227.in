  a  Z ZZ ZZZ  a ZZaZ    a   a Za a aaa  ZaZZ ZZaa Z aZ Z a aaZa aZ
ZZZ  ZZZaZ ZZZ aZa  aaZa  
 aZZaaaa ZZa a  ZaZZZ Za a aa a Z Z ZZa
  
ZZaZaaaZZ a ZaZZZaZa aZZZZ ZZa aZZaZZZaaaa Zaa ZZZZ Z a ZZZ  a a   Za
Za
ZaZ a  ZZZaa  ZZaZaa Za Za Z ZZ  aZ  ZZ  ZaZaZ Z ZZZaaaaZ
 aZaaa Za 
Z ZZZ  aZa aa
aZaaa  aa a
ZZ Z
a   ZZZZ aaaZaaaZZaa ZaZZaa a 
 Z     Z aa  Z  ZaaZ Z ZZ  ZZaZZ  aa a 
Z Zaaaa  aZ aaaZ  aZaaaZaa  aZaZa ZaZa ZZaZZZ aaa ZZaZ aZ  Z a a
ZaZ aa ZZa aaaZ  Za ZaZ    a
 a aa ZZZa Za ZZaZ ZZ   aZ  Z 
a ZZa aaZaaaZ Z ZaZa Z ZaZZa a a a    aZaaaZ   ZZZZZZZ  Za ZaZZ Za ZZ  Z  Z ZZ
a 
Za aaaa ZZa Z   aZ   Z ZZZ ZZZ  Za ZZaaaZZ  aZ Z ZZ aaaaZ 
