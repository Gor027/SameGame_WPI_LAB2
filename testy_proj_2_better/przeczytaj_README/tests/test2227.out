       Z
ZZ  a  Z
Zaa a  aZZ       Z
ZZZZa  ZaZZ Z    a
ZaZZaZ ZZZZZZ   ZaZa a   a
ZaaaZZZZZaaZZ   ZaZZ a aZa
aaZZaaaaaaaaaZZaZZZa ZZaZZ aaa       a
ZZZaZaaaZaaaZaZaaZaaZZZZZa ZZa    a  Za  Z     Z        Z
aZZZaaZZaaaaaaaZZaaaaZZZZa ZaZ Za Z aZa  a ZZ  Z aZ Za  Z
ZZaaaZaZZaaaaaaZZZaZZaaZaa ZZaaaZZaZZZaaaZaZa  ZaZZZaaZ Z    a
ZaZaaaZZZaZaaZZaaZaZaaaZaZ aaZZZZaZaZaZZZaaZaa aZZZZaZaaZ    a a
aaZZaaaZZaaZZaZZZaaZZZaZaZaaZaZZaaaaZaaZaaZZZZZZaZZZZZaaZZaZaaZa a Za
aaZaaaaaZZaaZaZZaZZaZZZZZZZZZZaaZaZZZaaaZZaZaZaZZZZZaaaaZaZZaZZaZaZZZZZZZ
